src/services/UserService.ts

```diff
@@ -65,39 +66,25 @@ export class UserService {
 
 			await validateOrReject(user).catch(e => Promise.reject(formatValidationErrors(e)));
 
-			const emailConfirmation = new EmailConfirmation();
-			try {
-				emailConfirmation.user = await entityManager.save(user);
-			} catch (error) {
```

Comment:
Bring the try/catch back so we can catch specific errors (email missing, missing info)